recipes/sqlitecpp/all/conanfile.py

```diff
@@ -78,11 +88,15 @@ def generate(self):
         tc.variables["SQLITECPP_BUILD_EXAMPLES"] = False
         tc.variables["SQLITECPP_BUILD_TESTS"] = False
         tc.variables["SQLITECPP_USE_STACK_PROTECTION"] = self.options.stack_protection
+        tc.variables["SQLITE_HAS_CODEC"] = self.options.get_safe("has_codec")
```

Comment:
```suggestion
        tc.variables["SQLITE_HAS_CODEC"] = self.options.get_safe("has_codec", False)
```
The `None` value is not handled correctly otherwise.